### PR TITLE
Additional header for Mobile Center telemetry

### PR DESCRIPTION
**Pull request by DavidStaheli**
Mobile Center requested this additional header for identifying Mobile Center Uploads from Team Services.  Corresponding changes in the Mobile Center Test task are waiting until the mobile-center CLI supports this.

### CODE REVIEW

File: Tasks/VSMobileCenterUpload/vsmobilecenterupload.ts [hunk 1]
```diff
@@ -118,7 +120,8 @@ function commitRelease(apiServer: string, apiVersion: string, appSlug: string, u
     tl.debug(`---- url: ${commitReleaseUrl}`);
     let headers = {
         "X-API-Token": token,
-        "User-Agent": userAgent
+        "User-Agent": userAgent,
+        "internal-request-source": "VSTS"
```
Comment (madhurig) [inline Tasks/VSMobileCenterUpload/vsmobilecenterupload.ts]: Is this help to mobile center indicate the upload is happening from VSTS? The "userAgent" string also contains "VSTS" but they will need to parse it.

Comment (AlexRukhlin) [inline Tasks/VSMobileCenterUpload/vsmobilecenterupload.ts]: Shouldn't we name this header "X-Internal-Request-Source"?

Comment (AlexRukhlin) [inline Tasks/VSMobileCenterUpload/vsmobilecenterupload.ts]: Shouldn't we name this header "X-Internal-Request-Source"?

Comment (DavidStaheli) [inline Tasks/VSMobileCenterUpload/vsmobilecenterupload.ts]: In addition to the **User-Agent** header that identifies VSTS, Mobile Center also uses the **internal-request-source** header to identify the source.  They had already defined this header and asked us to provide a value.

**Pull request merged**